src/main/java/com/twilio/http/Request.java

```diff
@@ -145,9 +144,9 @@ public URL constructURL() {
             URI uri = new URI(stringUri);
             return uri.toURL();
         } catch (final URISyntaxException e) {
-            throw new ApiException("Bad URI: " + stringUri, e);
+            throw new ApiException("Bad URI: " + e.getMessage());
```

Comment:
Why can't we log the `url`/`uri` as well?

Reply:
We can but it's already included at the end of the error message and will look redundant. 
Current output: `Bad URI: Illegal character in authority at index 7: http://{`
Add URI/URL:`Bad URI: http://{ Illegal character in authority at index 7: http://{`